packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -186,6 +193,18 @@ const { query, identifier, onInfiniteLoad } = usePaginatedQuery({
 })
 const { finalizeWizard } = useWorkspacesWizard()
 
+const canCreateProject = computed(
+  () => initialQueryResult.value?.workspaceBySlug?.permissions.canCreateProject
+)
+const canMoveProject = computed((): FullPermissionCheckResultFragment => {
+  // TODO: Until we have a real resolver
+  return {
+    authorized: isWorkspaceAdmin.value,
```

Comment:
Chuck's new permission check needs an update here to allow checking for general move permission, without having to specify a projectId